.github/workflows/on-push-do-doco.yml

```
@@ -0,0 +1,28 @@
+name: on-push-do-doco
+on:
+  # Only update the docs on default branch, to reduce the chance of conflicts
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run MarkdownSnippets
+      run: |
+        dotnet tool install --global MarkdownSnippets.Tool
+        mdsnippets ${GITHUB_WORKSPACE} \
+          --convention InPlaceOverwrite \
+          --exclude-directories 'cmake-build-*' \
```

```suggestion
          --exclude-directories 'recipes' \
```

Not sure if it works this way, but we need to exclude the folder with the recipes, just in case there is a `.md` file there. If modified it will modify the recipe revision and we would need to build it again.